client/src/modules/patients/documents/modals/documents.modal.html

```diff
@@ -1,3 +1,4 @@
+<script src="//wzrd.in/standalone/blob-util@latest"></script>
```

Comment:
What is this?  Why is it necessary?  Where it is used?

Reply:
oh! I should remove it